test/test_storage.bats

```diff
@@ -0,0 +1,106 @@
+setup() {
+    load 'lib/bats-support/load'
+    load 'lib/bats-assert/load'
+    load 'lib/common.bash'
+}
+
+teardown() {
+    ${KUBECTL} delete pod -l app=storage-test -n "$TARGET_NAMESPACE" --as system:admin
```

Comment:
We shouldn't need the `--as system:admin` in this test; creating pvcs and pods isn't a privileged operation (and we're not using it on any other tests). You should be operating in a namespace in which you already have permissions.